doc/usecases/elektra_web/UC_key_validation.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Key validation
+
+## Summary
+
+Title: Key validation
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User gets feedback about a key's contents.
+
+## Scenarios
+
+Precondition: Modifying keys
+Main success scenario: User edits a key's contents in a field that is
+  specifically formatted according to its metadata (e.g. number/date fields vs
+  string fields with regex validation). If the entered data passes the
+  validation, it is sent to the backend and saved to the key database.
+Alternative scenario: User enters data that fails the validation, the input
+  field is marked as invalid (e.g. with a red color and some information text)
+  and the data is NOT sent to the backend.
+Error scenario: Technical problems while persisting to the key database. The
```

Comment:
Is this a copy&paste problem? For me it is not clear when this use case applies. Isn't it before modifying keys and/or persisting keys? (So the user only typing into input fields without pressing ok).

Usually there is not only an invalid state but also a state where validation might pass if the user continues typing.